main-app/app/scripts/bingoTicket/bingoModel.js

```diff
@@ -6,32 +6,47 @@
             me.tickets = [];
             me.bingoStrip = [];
 
-            var clearOldTickets = function () {
-                me.tickets = [];
-                me.bingoStrip = [];
-            };
-
             me.getTicket = function (ticketFromServer) {
                 clearOldTickets();
                 generateTickets(ticketFromServer);
-                for (var i = 0; i < me.tickets.length; i++) {
+                for(var i = 0; i < me.tickets.length; i++){
                     createBingoTicketGrid();
                     splitingNumbersUp(me.tickets[i]);
```

Comment:
This isn't efficient in a loop, it will effect performance.
Split the ticket into an array once and only once. A sensible place to do this is at the poxy level, do that there isolates your game model from the way my API *happens* to represent the ticket. The API representation as a string is designed for service-efficiency purposes: sending as a (much more sensible) array will increase the message length by roughly a factor of 3: '011123....' ---> ['01','11','23']. Your model should never really see the "string" ticket and only deal with the array version.